modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -105,7 +105,30 @@ The following examples show situations that require configuring a network interf
 You cannot use `br-ex` bridge, an OVNKubernetes-managed Open vSwitch bridge, as the interface when configuring DNS resolvers.
 ====
 
-* Configure a static DNS for a network interface with an automatic IP configuration. Note that for this configuration, you must set the `auto-dns` parameter to `false`, so that the Kubernetes NMState Operator can store custom DNS settings for the network interface.
+* Configure a static DNS for a network interface with an automatic IP configuration. Note that when updating the `/etc/resolv.conf` file on a host node, you must not specify an interface (IPv4 or IPv6) in the `NodeNetworkConfigurationPolicy` (NNCP) manifest.
```

Comment:
> you must not specify an interface

This isn't strictly true. You _can_ specify an interface, you just don't need to anymore. I think we should soften this language to something like "you do not need to specify an interface".

Reply:
Fixed.